H2MLauncher.Core/H2MLauncher.Core.csproj

```diff
@@ -10,6 +10,17 @@
   <ItemGroup>
     <PackageReference Include="CommunityToolkit.Mvvm" Version="8.2.2" />
     <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```

Comment:
We prefer using `System.Text.Json` instead of `Newtonsoft.Json` it far surpasses newtonsofts performance and allocation while de/serialization. 

[System.Text.Json example](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to)